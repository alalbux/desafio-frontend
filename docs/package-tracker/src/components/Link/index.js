'use client';
import React from 'react';
import Link from '@mui/material/Link';

const StyledLink = ({ text, onClick, sx }) => {
  return (
    <Link
      variant="button"
      color="text.primary"
      onClick={() => onClick}
      sx={sx}
      underline="none"
      textTransform="math-auto"
      data-testid="link"
    >
      {text}
    </Link>
  );
};

export default StyledLink;
