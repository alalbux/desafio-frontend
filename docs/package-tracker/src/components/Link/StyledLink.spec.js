import React from 'react';
import { render } from '@testing-library/react';
import Link from '@/components/Link';

describe('Link', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Link />);
    expect(container).toMatchSnapshot();
  });
});
