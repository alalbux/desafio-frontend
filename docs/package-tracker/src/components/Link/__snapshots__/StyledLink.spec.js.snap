// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link matches the snapshot 1`] = `
<div>
  <a
    class="MuiTypography-root MuiTypography-button MuiLink-root MuiLink-underlineNone css-17f0go1-MuiTypography-root-MuiLink-root"
    data-testid="link"
  />
</div>
`;
